tests/xtimer_cli/tests/Xtimer.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2018 Kevin Weiss <kevin.weiss@haw-hamburg.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+"""@package PyToAPI
+This module handles parsing of information from RIOT xtimer_cli test.
+"""
+import logging
+
+from riot_pal import DutShell
+from robot.version import get_version
```

Comment:
Also this should be separated out into a robot independent interface then a simple wrapper for robot.